packages/neutron-vesting-lp-vault/src/types.rs

```diff
@@ -0,0 +1,80 @@
+use crate::error::{ContractError, ContractResult};
+use cosmwasm_std::Addr;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, JsonSchema, Debug)]
+pub struct Config {
+    pub name: String,
+    pub description: String,
+    pub atom_vesting_lp_contract: Addr,
+    pub atom_oracle_contract: Addr,
+    pub usdc_vesting_lp_contract: Addr,
+    pub usdc_oracle_contract: Addr,
+    pub owner: Addr,
+    pub manager: Option<Addr>,
```

Comment:
What's even the purpose of a manager here?

Reply:
uhh, it is a leftover from a sketched lockdrop-vault contract. Maybe I should remove a manager from a contract?

Reply:
currently the manager is capable of updating the vault config. don't we want it?

Reply:
Moved to #54